### PR TITLE
Modified Config and ConfigTest to use File.pathSeparator to allow for multi-platform support

### PR DESCRIPTION
**Pull request by traviswinter**
Realized today the tests were failing locally for me - took me longer then I would like to admit to realize it was due to the multi-file support added recently.

### CODE REVIEW

**Comment by rohanKanojia** [comment]
ok to test

**Comment by rohanKanojia** [comment]
[merge]

File: kubernetes-client/src/test/java/io/fabric8/kubernetes/client/ConfigTest.java [hunk 1]
```diff
@@ -257,7 +257,8 @@ public void testWithKubeConfigAndOverrideContext() {
 
   @Test
   public void testWithMultipleKubeConfigAndOverrideContext() {
-    System.setProperty(Config.KUBERNETES_KUBECONFIG_FILE, TEST_KUBECONFIG_FILE + ":some-other-file");
+    System.setProperty(Config.KUBERNETES_KUBECONFIG_FILE, TEST_KUBECONFIG_FILE + File.pathSeparator + "some-other-file");
```
Comment (iocanel) [inline kubernetes-client/src/test/java/io/fabric8/kubernetes/client/ConfigTest.java]: This doesn't seem right to me. It should use File.sepsrator instead.

Comment (rohanKanojia) [reply kubernetes-client/src/test/java/io/fabric8/kubernetes/client/ConfigTest.java]: Actually, this delimiter is for separating multiple files here. It should be `File.pathSeparator` in my opinion. @oscerd Wdyt????

Comment (iocanel) [reply kubernetes-client/src/test/java/io/fabric8/kubernetes/client/ConfigTest.java]: @rohanKanojia: right. Ignore my comment.

Comment (oscerd) [reply kubernetes-client/src/test/java/io/fabric8/kubernetes/client/ConfigTest.java]: I'm late :-)

**Pull request merged**